### DISCUSSION TITLE
prevent !important in css from ui

### DISCUSSION THREAD

**Discussion by ronaldohoch** [GrapesJS/grapesjs#5336] [Q&A] [answered]
Hi @artf, how you doing?
Is there a way to prevent it to happen?

![image](https://github.com/GrapesJS/grapesjs/assets/2287371/abb98111-2b30-40a8-8372-b96eb76ef391)

**Comment by artf** [comment, marked as answer]
Hi @ronaldohoch well that doesn't happen by default, so try to elaborate better your use case

**Reply by ronaldohoch** [reply]
I don't know how the user made this :( 
Thank you.
